SensioLabs/AnsiConverter/AnsiToHtmlConverter.php

```diff
@@ -41,7 +41,7 @@ public function convert($text)
     {
         // remove cursor movement sequences
         $text = preg_replace('#\e\[(K|s|u|2J|2K|\d+(A|B|C|D|E|F|G|J|K|S|T)|\d+;\d+(H|f))#', '', $text);
-        $text = htmlspecialchars($text, ENT_QUOTES, $this->charset);
+        $text = htmlspecialchars($text, ENT_QUOTES | ENT_SUBSTITUTE, $this->charset);
```

Comment:
I would prefer to conditionally add this FLAG when supported instead of bumping the requirements. Something like `(PHP_VERSION_ID >= 50400 ? ENT_QUOTES | ENT_SUBSTITUTE : ENT_QUOTES)`